packages/client/src/pages/Profile/elements/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,43 @@
+import { AvatarImage } from '@assets/images'
+import styles from './avatar.module.css'
+import { ChangeEvent, FC, useState } from 'react'
+import clsx from 'clsx'
+import { AvatarProps } from './type'
+import { updateAvatar } from '@services/profile'
+
+export const Avatar: FC<AvatarProps> = ({ avatar }) => {
+  const [avatarProfile, setAvatar] = useState<string>(avatar ? avatar : '')
```

Comment:
[Исправление]
Можно сократить запись, если использовать оператор `??`, логика не изменится, будет пустая строка, если `avatar` - `null` или `undefined`, в остальных случаях - `avatar`
```suggestion
  const [avatarProfile, setAvatar] = useState<string>(avatar ?? '')
```